format/private/formatter_binary.bzl

```diff
@@ -18,6 +18,7 @@ _TOOLS = {
     "sql": "prettier-sql",
     "sh": "shfmt",
     "protobuf": "buf",
+    "clangformat": "clang-format",
```

Comment:
in other cases we tried to have the user-visible part (the name of the attribute, so the key in this dict) be the name of the language, rather than the name of a tool. Should we make it `cpp` or is that confusing?

Reply:
Yes or `cc`?

Reply:
I went with `cc` as this matches `rules_cc`